### ISSUE TITLE
[BUG] Reload command not affecting TAGS

### ISSUE THREAD

**Issue by ElMoha943** [LaBoulangerie/TownyChatRemake#2]
/ta reload chat is not affecting chat format, is there another command to reload the chat?

**Comment by ElMoha943** [comment]
nvm i had an error in config, but now my chat looks like this 
![imagen](https://user-images.githubusercontent.com/25410859/166943179-038ff641-84ca-4d6f-a126-323a98ef62e1.png)

```
channels:
    global:
        name: "Global Chat"
        format: "<global_prefix> » &f<cityplate><rankplate><nameplate> &7:{vault_suffix}<message>"
        command: "gc"

    town:
        name: "Town Chat"
        format: "<town_prefix> <rank> <username> <aqua>> <message>"
        spy_format: "<spy_prefix> <town_prefix> <town> <username> <dark_purple>> <message>"
        command: "tc"

    nation:
        name: "Nation Chat"
        format: "<nation_prefix> <town><rank> <username> <gold>> <message>"
        spy_format: "<spy_prefix> <nation_prefix> <nation> <username> <purple>> <message>"
        command: "nc"

# You can use PAPI's placeholders and MiniMessage syntax here too!
tags:
    nameplate: "<hover:show_text:'Click para abrir el perfil de este usuario'><click:run_command:profile %player_name%>%player_displayname%</click></hover>"
    rankplate: "<hover:show_text:'&7Click para abrir la tienda.'><click:run_command:tienda>%vault_prefix%</click></hover>"
    cityplate: "<hover:show_text:'&7Ciudad: %townyadvanced_town%\n&7Nacion: %townyadvanced_nation%'><click:run_command:resident %player_name%>%conditionaltext_setting-rank%</click></hover>"
    global_prefix: "<hover:show_text:'&7Canal: Global'>&7[&8GC&7]</hover>"
    town_prefix: "<hover:show_text:'&7Canal: Ciudad'>&7[&3TC&7]</hover>"
    nation_prefix: "<hover:show_text:'&7Canal: Nacion'>&7[&ENC&7]</hover>"
    spy_prefix: "<hover:show_text:'&7Canal: Spy'>&7[&CSP&7]</hover>"
```

**Comment by ElMoha943** [comment]
~Also how do i use HEX colors in chat format?~ Would be easier with & instead of the <> format, will be nice to add support for it

Edit: figured it out.

**Comment by ElMoha943** [comment]
Also would be nice that if i do /tc without text it changes my channel to town, like the original townychat.

**Comment by ElMoha943** [comment]
Format reload reload was fixed but not tags, i dont know if is a bug or is not possible to update tags with a reload.

**Comment by GroobleDierne** [comment]
It should be fixed by 390ab410e0ef7dc938ef5e962a812cd20f53d89c, open another issue if it isn't.
We will make a new release quickly.
